test/integration/stateView/document/fetchDocumentsFactory.spec.js

```diff
@@ -113,4 +135,22 @@ describe('fetchDocumentsFactory', () => {
 
     expect(result).to.deep.equal([]);
   });
+
+  it('should throw InvalidQueryError if searching by non indexed fields', async () => {
```

Comment:
We need to test one more case with the wrong type.

Reply:
We already have it. `should return empty array if type does not exist`